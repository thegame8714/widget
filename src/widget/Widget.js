import React from 'react'

const Widget = () => <div>Widget</div>

export default Widget
