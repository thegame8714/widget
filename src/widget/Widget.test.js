import React from 'react'
import Widget from './Widget'
import renderer from 'react-test-renderer'

describe('Widget', () => {
  describe('Application', () => {
    it('should render the application', () => {
      const App = renderer.create(<Widget />).toJSON()

      expect(App).toMatchSnapshot()
    })
  })
})
