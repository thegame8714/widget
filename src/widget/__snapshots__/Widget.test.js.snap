// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Widget Application should render the application 1`] = `
<div>
  Widget
</div>
`;
